### PR TITLE
Update to Notebook 7.0.8

### PR DESCRIPTION
**Pull request by jtpio**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jtpio** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
-{% set version = "7.0.7" %}
-{% set lab_version = ">=4.0.7,<5" %}
+{% set version = "7.0.8" %}
+{% set lab_version = ">=4.0.7,<4.1" %}
```
Comment (jtpio) [inline recipe/meta.yaml:2]: Relevant upstream PR: https://github.com/jupyter/notebook/pull/7249

Comment (bollwyvl) [reply recipe/meta.yaml:2]: prolly needs a `repodata-patches` PR, otherwise solver will just pull older ones.

Comment (bollwyvl) [reply recipe/meta.yaml:2]: (but this should land here first)

**Review by bollwyvl — APPROVED** [review]
LGTM, will do the `repodata-patches` PR after this is merged

File: recipe/meta.yaml (hunk 1 above)
Comment (bollwyvl) [reply recipe/meta.yaml:2]: ready to go, but still in draft: https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/662

**Pull request merged**